packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.html

```diff
@@ -1,9 +1,14 @@
 <template>
-  <li class="slds-m-around_small slds-grid">
-    <div class="slds-col_bump-right">
-      <input type="checkbox" onchange={onSelect} />
-      <span class="slds-m-horizontal_medium nowrap">{item.value}</span>
+  <li class="slds-var-m-around_small slds-grid " onclick={facetValueClick}>
+    <div class="slds-col_bump-right ">
+      <template if:true={isChecked}>
+        <input type="checkbox" checked/>
```

Comment:
Could you pass the boolean value to the `checked` attribute instead of using templates?

    <input type="checkbox" checked={isChecked} />

Reply:
checked is not an attribute where you pass `true` or `false`. if the attribute is present on the HTML, it means the default state of the checkbox is checked, if the attribute is not present on the checkbox, it means it isn't checked.

so `checked="false"` doesn't do what you expect :) 

This is why I have the template if:true

Reply:
:thinking: I did a quick test in a separate LWC project and passing a boolean value to the `checked` attribute works just fine. Maybe there is something special with the facet values that makes it different...

Reply:
`true` works, but did you try passing false to "uncheck" the checkbox? ;)

Reply:
`false` is working as well for me. I guess the LWC implementation of the `input` is different than the link you posted.
But, let's keep the templates if you prefer.

Reply:
If false works then it's the browser that's "fixing" the standard for you.

Find any html checkbox somewhere, edit it in your browser to put checked="false" and it should not uncheck.

Or go to the link I sent there's a live code example.